### DISCUSSION TITLE
Is there a way to set "clear" and "done" button?

### DISCUSSION THREAD

**Discussion by riofrank** [mmazzarolo/react-native-modal-datetime-picker#631] [Q&A] [answered]
Hi,

Is there a way to set "clear" and "done" button as below?
![image](https://user-images.githubusercontent.com/37997033/155868843-37d700c8-7c72-406e-ba89-d4c454547074.png)

Thank you.

**Comment by mmazzarolo** [comment, marked as answer]
Hey! 
For the iOS one it might just be easier to build it from scratch (not sure why you'd want to do it though since it's not looking like the native one). 
For Android, there's nothing we can do — that UI is the old-style Android picker, and it's controlled directly by the native component (which we don't have control on).
